renderer/components/editor/options/export-select.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const OptionType = PropTypes.shape({
+  label: PropTypes.string.isRequired,
+  value: PropTypes.string.isRequired
+});
+
+const Chevron = () => (
+  <svg xmlns="http://www.w3.org/2000/svg" width="10" height="5" viewBox="0 0 10 5" className="c-select__toggle-icon" style={{marginLeft: '8px'}}>
```

Comment:
This exists already under `/renderer/vectors/dropdown-arrow`. You can import it as `import {DropdownArrow} from '../../../vectors'` (the vectors index)